Services/CRUD/CrudManager.php

```diff
@@ -413,11 +415,16 @@ public function setData(CrudEntityInterface $entity, array $data = array())
      */
     public function save(CrudEntityInterface $entity)
     {
-        $violations = $this->validate($entity);
-        if ($violations instanceof ConstraintViolationList) {
-            throw new ValidationFailedException($violations);
+        if ($entity instanceof ValidatableInterface) {
+            $validation = $this->validate($entity);
+            if ($validation instanceof ConstraintViolationListInterface) {
+                $entity->getViolations()->addAll($validation);
+            }
+            if ($entity->getViolations()->count()) {
+                throw new ValidationFailedException($entity->getViolations());
+            }
         }
-        $this->flush($entity);
+        $this->flush();
```

Comment:
I have entities that are updated in relation to each other. Flushing only the given entity will break this. @RuslanZavacky do you know the original intent of flushing only $entity?